Makefile

```diff
@@ -25,6 +25,12 @@ export BUILD_HARNESS_AUTO_INIT := false
 # Debug should not be defaulted to a value because some cli consider any value as `true` (e.g. helm)
 export DEBUG ?=
 
+# Support for colorized output
```

Comment:
Note that this is already in `Makefile.helpers`. Can you put that there?
https://github.com/oneconcern/build-harness/blob/de64b5358dccb5a0cdc03b451b3dcc14e0b5e4c1/Makefile.helpers#L12

Reply:
Done. I kept the uppercase names if that's alright with you.